src/otx/core/data/module.py

```diff
@@ -62,18 +62,15 @@ def __init__(
         auto_num_workers: bool = False,
         device: DeviceType = DeviceType.auto,
         input_size: int | tuple[int, int] | None = None,
+        adaptive_input_size: Literal["auto", "downscale", "none"] = "none",
+        input_size_multiplier: int = 1,
```

Comment:
When is this value used?

In my opinion, it's hard for people using the API to understand the functionality at a glance by the name of the argument. Is it possible to add a docstring for the `OTXDataModule`?

Reply:
Some models require input_size to be multiple of specific value (e.g. yolox should use 32xN input_size). I'll update docstring for this.

Reply:
done https://github.com/openvinotoolkit/training_extensions/pull/3788/commits/0adb7ea54bb57b761f75b423167b45772d08d416